mods/ra/maps/training-camp/map.yaml

```diff
@@ -1058,9 +1058,10 @@ VoxelSequences:
 
 Weapons:
 	CrateNuke:
-		Warhead@areanuke2: PerCellDamage
+		Warhead@areanuke2: SpreadDamage
 			Damage: 250
-			Size: 4
+			Spread: 1c0
+			Falloff: 100, 100, 100, 100, 100, 0
```

Comment:
Demo trucks use the [```MiniNuke```](https://github.com/OpenRA/OpenRA/blob/e875240ca35d537d484a0ec16868c3b6145f6dc1/mods/ra/rules/vehicles.yaml#L653) instead of the ```CrateNuke```.